### PR TITLE
Fix issues pointed out by go report card

### PR DESCRIPTION
**Pull request by nmccrory**
I ran into a lot of issues with my fork after drud/ghlabel became public. I ended up just having to delete my old fork and create a new one. ):

### CODE REVIEW

File: cmd/ghlabel/main.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func NewClient() *Client {
 func (c *Client) ListByUser() error {
 	if ApplyLabels {
 		printCommitHeader()
-	}else{
+	} else {
```
Comment (rfay) [inline cmd/ghlabel/main.go]: This implies that your circle tests weren't doing gofmt? I'd recommend making sure your circle tests do a `make gometalinter` and/or `make staticcheck` please

File: cmd/ghlabel/main.go [hunk 2]
```diff
@@ -242,9 +242,9 @@ func printCommitHeader() {
 func printPreviewHeader() {
 	color.Yellow("Running in preview mode...\n\n")
 	fmt.Println("Description:")
-	fmt.Println("  View currently staged label updates.\n")
+	fmt.Print("\n  View currently staged label updates.\n")
```
Comment (rfay) [inline cmd/ghlabel/main.go]: No need to change this, but why not one big static string instead of 4 print statements?

Comment (nmccrory) [reply cmd/ghlabel/main.go]: In this case I prefer breaking it up for readability. This is identical to the way it looks when printed to the console.

**Comment by nmccrory** [comment]
`make gometalinter` is now integrated into our Circle config.

**Comment by cyberswat** [comment]
Adding a note to link this to #3

**Comment by rfay** [comment]
@nmccrory But... you don't have circle testing visible at this point. You'll need to enable circle testing. I'm betting you have it enabled on your fork, but not on drud/ghlabel?

**Comment by rfay** [comment]
It also does not *pass* `make gometalinter` - it fails.

Please consider upgrading build-tools as well, since you're here. That will give you a somewhat better staticcheck and maybe gometalinter.

**Comment by nmccrory** [comment]
@rfay I upgraded my build tools to v1.5.0 but they've always been ignored by git so I don't quite see a real way to push them into a PR.

Please correct me if I am missing something.

**Comment by rfay** [comment]
When you upgrade build-tools it makes a commit. Try a `git log` to see. I recommended doing it as a separate PR so it would not clog up the reviewing of this one, but all is good.

This PR still isn't doing circle builds.

**Comment by rfay** [comment]
Yay, circle build is working, now to fix gometalinter issues. How did you get circle build to start?

**Comment by nmccrory** [comment]
Tanner helped me by telling me that there was a quirk I could use to fix the issue. I simply unfollowed the project in Circle then pushed a "nudge" commit and it kicked off the build. Oh the quirks of CircleCI...

**Comment by nmccrory** [comment]
Once this is merged I can pull the latest changes from upstream and push them into my other branches with open PRs.

**Pull request merged**